IO.Eventuate.Tram.Tests/TestHelpers/TestServicesHost.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using IO.Eventuate.Tram.Local.Kafka.Consumer;
+using IO.Eventuate.Tram.Database;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+
+namespace IO.Eventuate.Tram.Tests.TestHelpers
+{
+    public class TestServicesHost
+    {
+        public string _aggregateType = "test";
+        public void SetAggregateType(string aggregateType)
+        {
+            _aggregateType = aggregateType;
+        }
+        public IHost SetUpHost()
+        {
+            var _host = new HostBuilder()
+           .ConfigureServices((hostContext, services) =>
+           {
+               // Logging 
+               services.AddLogging(builder =>
+               {
+                   builder.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Debug);
+                   builder.AddConsole();
+                   builder.AddDebug();
+               });
+               // DBContext
+               services.AddDbContext<EventuateTramDbContext>((provider, o) =>
+               {
+                   o.UseSqlServer(TestSettings.EventuateTramDbConnection);
+               });
+               // Kafka Transport
+               services.AddEventuateTramSqlKafkaTransport(TestSettings.EventuateTramDbSchema, TestSettings.KafkaBootstrapServers, EventuateKafkaConsumerConfigurationProperties.Empty(),
+                   (provider, o) =>
+                   {
+                       o.UseSqlServer(TestSettings.EventuateTramDbConnection);
+                   });
+               // Publisher
```

Comment:
MessageTest doesn't need the events configuration. See above comment about having separate Host classes

Reply:
Added Separate Service host in Unit tests itself